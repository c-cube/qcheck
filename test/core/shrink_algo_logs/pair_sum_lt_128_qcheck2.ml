open QCheck2

let pair_sum_lt_128 =
  Test.make ~name:"pairs sum to less than 128"
    Gen.(pair (pint ~origin:0) (pint ~origin:0))
    (Log.shrinks Print.(pair int int) @@
     fun (i,j) -> i+j<128)

let rand = Random.State.make [|1234|];;
try Test.check_exn ~rand pair_sum_lt_128 with Test.Test_fail _ -> ()
