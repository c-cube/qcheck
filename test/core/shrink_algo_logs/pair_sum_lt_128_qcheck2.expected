holds (1526421322787444331, 4437769643960039450)
holds (3367339764960236310, 4222330705308310169)
holds (2790778446709483554, 2587213195390812545)
holds (3066594532144543221, 2606114362798169773)
fails (617863026503364673, 924354758750856816)
fails (0, 924354758750856816)
holds (0, 0)
fails (0, 462177379375428408)
holds (0, 0)
fails (0, 231088689687714204)
holds (0, 0)
fails (0, 115544344843857102)
holds (0, 0)
fails (0, 57772172421928551)
holds (0, 0)
fails (0, 28886086210964275)
holds (0, 0)
fails (0, 14443043105482137)
holds (0, 0)
fails (0, 7221521552741068)
holds (0, 0)
fails (0, 3610760776370534)
holds (0, 0)
fails (0, 1805380388185267)
holds (0, 0)
fails (0, 902690194092633)
holds (0, 0)
fails (0, 451345097046316)
holds (0, 0)
fails (0, 225672548523158)
holds (0, 0)
fails (0, 112836274261579)
holds (0, 0)
fails (0, 56418137130789)
holds (0, 0)
fails (0, 28209068565394)
holds (0, 0)
fails (0, 14104534282697)
holds (0, 0)
fails (0, 7052267141348)
holds (0, 0)
fails (0, 3526133570674)
holds (0, 0)
fails (0, 1763066785337)
holds (0, 0)
fails (0, 881533392668)
holds (0, 0)
fails (0, 440766696334)
holds (0, 0)
fails (0, 220383348167)
holds (0, 0)
fails (0, 110191674083)
holds (0, 0)
fails (0, 55095837041)
holds (0, 0)
fails (0, 27547918520)
holds (0, 0)
fails (0, 13773959260)
holds (0, 0)
fails (0, 6886979630)
holds (0, 0)
fails (0, 3443489815)
holds (0, 0)
fails (0, 1721744907)
holds (0, 0)
fails (0, 860872453)
holds (0, 0)
fails (0, 430436226)
holds (0, 0)
fails (0, 215218113)
holds (0, 0)
fails (0, 107609056)
holds (0, 0)
fails (0, 53804528)
holds (0, 0)
fails (0, 26902264)
holds (0, 0)
fails (0, 13451132)
holds (0, 0)
fails (0, 6725566)
holds (0, 0)
fails (0, 3362783)
holds (0, 0)
fails (0, 1681391)
holds (0, 0)
fails (0, 840695)
holds (0, 0)
fails (0, 420347)
holds (0, 0)
fails (0, 210173)
holds (0, 0)
fails (0, 105086)
holds (0, 0)
fails (0, 52543)
holds (0, 0)
fails (0, 26271)
holds (0, 0)
fails (0, 13135)
holds (0, 0)
fails (0, 6567)
holds (0, 0)
fails (0, 3283)
holds (0, 0)
fails (0, 1641)
holds (0, 0)
fails (0, 820)
holds (0, 0)
fails (0, 410)
holds (0, 0)
fails (0, 205)
holds (0, 0)
holds (0, 102)
fails (0, 153)
holds (0, 0)
holds (0, 76)
holds (0, 114)
fails (0, 133)
holds (0, 0)
holds (0, 66)
holds (0, 99)
holds (0, 116)
holds (0, 124)
fails (0, 128)
holds (0, 0)
holds (0, 64)
holds (0, 96)
holds (0, 112)
holds (0, 120)
holds (0, 124)
holds (0, 126)
holds (0, 127)
